packages/wonder-blocks-modal/components/modal-dialog.js

```diff
@@ -17,7 +18,7 @@ type Props = {|
 export default class ModalDialog extends React.Component<Props> {
     render() {
         const {style, children} = this.props;
-        const contextValue = {
+        const contextValue: MediaLayoutContextValue = {
```

Comment:
Nice! (Not convinced we need the word `Value` on the end of that type, but I'm not holding a strong opinion on that).

Reply:
I'm not super happy about the name either but `MediaLayoutContext` is already taken. 😞